libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -385,11 +382,12 @@ public function handleStorage(Element $storage)
 					{
 						case 'csv':
 							$handler = '\Joomla\ORM\Storage\CsvProvider';
-							$param   = $parts[1];
+							break;
+						case 'orm':
+							$handler = '\Joomla\ORM\Storage\Doctrine\DoctrineProvider';
```

Comment:
This would mean, if I want to have the password in a different file I can't. There will be ways where it needs to be more flexible with parameters for sure wich will not fit into the ORM scheme. For example when when I have my own api provider which should work for multiple entities and I want to define which server it needs to access per entity to get the data from. Then I can't, except when the provider has hardcoded every entity url inside the code. Sometimes it is not bad to have some kind of flexibility.